### PR TITLE
{vis}[foss/2018a] scikit-image 0.13.1 + Pillow 5.0 0 w/ Python 3.6.4

### PR DESCRIPTION
**Pull request by RvDijk**
dependencies on ~~#5810~~ (X11), ~~#5743~~ (matplotlib)

### CODE REVIEW

File: easybuild/easyconfigs/s/scikit-image/scikit-image-0.13.1-foss-2018a-Python-3.6.4.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+dependencies = [
+    ('Python', '3.6.4'),
+    ('Qhull', '2015.2'),
+    ('matplotlib', '2.1.2', versionsuffix),
+    ('Pillow', '5.0.0', versionsuffix),
+]
+
+exts_list = [
+    ('networkx', '2.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/n/networkx', 'https://github.com/networkx/networkx/archive/'],
```
Comment (boegel) [inline easybuild/easyconfigs/s/scikit-image/scikit-image-0.13.1-foss-2018a-Python-3.6.4.eb]: @RvDijk Why the additional source URL for `networkx`, not needed?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2666.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1d5174f0e5091a3c42334ac0a979b32b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2034.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/592a423bf4d4cc4d4ade40d9b6d40518 for a full test report.

File: easybuild/easyconfigs/s/scikit-image/scikit-image-0.13.1-foss-2018a-Python-3.6.4.eb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+dependencies = [
+    ('Python', '3.6.4'),
+    ('Qhull', '2015.2'),
+    ('matplotlib', '2.1.2', versionsuffix),
+    ('Pillow', '5.0.0', versionsuffix),
+]
+
+exts_list = [
+    ('networkx', '2.1', {
+        'source_urls': ['https://github.com/networkx/networkx/archive/'],
```
Comment (boegel) [inline easybuild/easyconfigs/s/scikit-image/scikit-image-0.13.1-foss-2018a-Python-3.6.4.eb]: @RvDijk In existing easyconfigs we use `https://pypi.python.org/packages/source/n/networkx`, why the switch?

Comment (RvDijk) [reply easybuild/easyconfigs/s/scikit-image/scikit-image-0.13.1-foss-2018a-Python-3.6.4.eb]: @boegel at the time of installing I seemed to have received a wrong package, causing the installation to fail. However I can't reproduce this error anymore. I'll change back to the original source..

**Comment by boegel** [comment]
Going in, thanks @RvDijk!

**Pull request merged**